internal/db/update/schema.go

```diff
@@ -37,8 +37,8 @@ func (s *SchemaUpdate) Check(check schema.Check) {
 	s.check = check
 }
 
-func (s *SchemaUpdate) Version() int {
-	return len(s.updates)
+func (s *SchemaUpdate) Version() (int, int) {
```

Comment:
Could we have a function prototype like `func (s *SchemaUpdate) Version() (internal int, external int)` to distinguish the return values (useful for `gopls` in VSCode when hovering the code)